src/odoc.ml

```diff
@@ -71,10 +71,15 @@ module Dep = struct
 
   let alias = Alias.make ".odoc-all"
 
-  let deps ctx requires =
+  let deps ctx pkg requires =
     Build.of_result_map requires ~f:(fun libs ->
       Build.deps (
-        List.fold_left libs ~init:Dep.Set.empty ~f:(fun acc (lib : Lib.t) ->
+        let init =
+          match pkg with
+          | Some p -> Dep.Set.singleton (Dep.alias (alias ~dir:(Paths.odocs ctx (Pkg p))))
```

Comment:
This line looks a bit long. We stick to 80 chars in dune.

Reply:
Fixed